src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -152,12 +165,16 @@
     <UpdateNuGetConfigPackageSourcesMappings
       NuGetConfigFile="$(NuGetConfigFile)"
       BuildWithOnlineFeeds="$(DotNetBuildWithOnlineFeeds)"
-      SourceBuildSources="$(SourceBuildSources)" />
+      SourceBuildSources="@(SourceBuildSources)" />
 
     <MakeDir Directories="$(BaseIntermediateOutputPath)" />
     <Touch Files="$(BaseIntermediateOutputPath)UpdateNuGetConfig.complete" AlwaysCreate="true">
       <Output TaskParameter="TouchedFiles" ItemName="FileWrites" />
     </Touch>
+
+    <ItemGroup>
+      <EmbedInBinlog Include="$(NuGetConfigFile)" />
+  </ItemGroup>
```

Comment:
indentation is off:

```suggestion
    </ItemGroup>
```

Reply:
Fixed in https://github.com/dotnet/installer/pull/18557/commits/8b3f3b271825f5c273d951a1c95160516e6ad51e